CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Ephemeral Namespace Operator Local Development
+
+### Purpose
+The purpose of this doc is to deploy the Ephemeral Namespace Operator (ENO) in Minikube for local development.  
+
+### Requirements
+- Clone the [Clowder](https://github.com/RedHatInsights/clowder) repository
+- Install [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- Login credentials for [Quay](https://quay.io)
+- Install [Bonfire](https://pypi.org/project/crc-bonfire/)
+
+### Setup Minikube
+```
+minikube start --cpus 4 --disk-size 36GB --memory 8192MB --addons registry --addons ingress --addons metrics-server
+```
+
+### Retrieve Quay Pull Secret
```

Comment:
Can you please add a section here explaining why the Quay pull secret is necessary here?